data/style.css

```
@@ -6,6 +6,14 @@ body {
 	padding: 0 16px ;
 	margin-bottom: 500px ;
 	font-family: sans-serif ;
+	scrollbar-color: gray #151515;
+}
+::-webkit-scrollbar {
+	width: .66vw;
+	background-color: #151515;
+}
+::-webkit-scrollbar-thumb {
+	background-color: gray;
 }
 
```

```suggestion
::-webkit-scrollbar-track:vertical:disabled:corner-present {
margin-bottom: 0px;
}
::-webkit-scrollbar-corner{
background-color: #151515;
}
```
It is necessary for OP to style the CORNER for dark mode, cus by deafult it is white.
The top bit of code I added also will actually hide the corner unless there is a horizontal bar (when zoomed in), I prefer this. Luke may disagree and remove that top bit I dunno.

---

Additionaly, if Luke wants a slightly rounded scrollbar:

::-webkit-scrollbar-thumb{      **border-radius: 5px;**  ...

---

IMHO disabling the corner is anything but intuitively comprehensive. I'm more in the favor of styling the corner.

But anyway: Do we need that? The site is perfectly responsive. I checked and found no situation when horizontal scrollbars would appear...

It's a balance of keeping the stylesheet small and comprehensive vs not having to cover that case if it happens in the future. I'm in favor of the former. Do you think it is needed?

---

> Additionaly, for a slightly rounded scrollbar:
Done. Thanks for the suggestion!

---

> IMHO disabling the corner is anything but intuitively comprehensive. I'm more in the favor of styling the corner.
> 
> But anyway: Do we need that? The site is perfectly responsive. I checked and found no situation when horizontal scrollbars would appear...
> 
> It's a balance of keeping the stylesheet small and comprehensive vs not having to cover that case if it happens in the future. I'm in favor of the former. Do you think it is needed?

Alright, as long as it looks good from your end (in both light and dark modes).